pandas/io/json/_json.py

```diff
@@ -208,6 +208,16 @@ def to_json(
         indent=indent,
     ).write()
 
+    if orient == "split" and isinstance(obj, DataFrame) and isinstance(obj.columns, MultiIndex):
```

Comment:
This should probably be handled in the writer, not after the json string is written

Reply:
@mroeschke good point, I have just made a new commit with this suggested change; I moved the code to the FrameWriter class